src/ngraph/node.cpp

```diff
@@ -787,7 +751,7 @@ NodeVector ngraph::as_node_vector(const OutputVector& values)
     NodeVector node_vector;
     for (auto& value : values)
     {
-        node_vector.push_back(value.as_single_output_node());
+        node_vector.push_back(value.get_node_shared_ptr());
```

Comment:
This is incorrect. Instead, `as_node_vector` callers need to be changed to work with `OutputVector`. There probably aren't many left.